requirements.txt

```diff
@@ -6,3 +6,4 @@ pyparsing>=2.2.0
 python-Levenshtein
 termcolor
 wcwidth
+pre-commit
```

Comment:
This will work, but `requirements.txt` contains the _runtime_ dependencies, whereas `pre-commit` as a development dependency. I see two options here:
* Declare `pre-commit` as a development dependency directly in [`setup.py`](https://github.com/facebookincubator/python-nubia/blob/master/setup.py)
* Introduce a new `requirements-dev.txt` file and make [`setup.py`](https://github.com/facebookincubator/python-nubia/blob/master/setup.py) use it